client/coral-embed-stream/src/Embed.js

```diff
@@ -158,6 +159,11 @@ class Embed extends Component {
               clearNotification={this.props.clearNotification}
               notification={{text: null}}
             />
+          <LoadMore
+            asset_id={asset.id}
+            comments={asset.comments}
+            moreComments={asset.commentCount > asset.comments.length}
```

Comment:
@okbel Once you hit "load more" the coments array can be greater than 5, so this is essentially saying "is the commentCount greater than the number of comments currently loaded?"